akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -292,6 +312,14 @@ object Flow extends FlowApply {
   private def shape[I, O](name: String): FlowShape[I, O] = FlowShape(Inlet(name + ".in"), Outlet(name + ".out"))
 
   /**
+   * Creates a Flow from a Reactive Streams [[org.reactivestreams.Processor]]
+   */
+  def apply[I, O](processorFactory: () ⇒ Processor[I, O]): Flow[I, O, Unit] = {
```

Comment:
The downside of overloading here is that now we have wasted using Function1 never to be used again in Flow.apply.